services/bettersocial/post/createPostV3.js

```diff
@@ -330,13 +338,13 @@ const BetterSocialCreatePostV3 = async (req, isAnonimous = true) => {
         anonUserInfoEmojiName: body?.anon_user_info?.emoji_name
       });
     }
-
     await processSendSystemMessage(
       filteredTopics,
       req?.userId,
       user?.username,
       body?.with_system_message,
-      isAnonimous
+      isAnonimous,
+      body?.duration_feed
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `processSendSystemMessage` has been updated with a new parameter `body?.duration_feed`, but it's unclear what this parameter does from the context provided. Make sure that the function definition of `processSendSystemMessage` is updated accordingly to handle this new argument. If this parameter is optional, ensure that there are checks in place within the function to handle cases when it is not provided.


<!-- This is an auto-generated comment by OSS CodeRabbit -->